pkg/controllers/csi/metadata/models.go

```diff
@@ -37,15 +37,15 @@ type OSMount struct {
 	VolumeMeta VolumeMeta
 	TimeStampedModel
 	TenantUUID    string `gorm:"primaryKey" json:"tenantUUID,omitempty"`
-	VolumeMetaID  string
+	VolumeMetaID  string `gorm:"not null"`
 	Location      string `gorm:"not null"`
 	MountAttempts int64  `gorm:"not null"`
 }
 
 // AppMount keeps track of our mounts to user applications, where we provide the codemodules.
 type AppMount struct {
 	VolumeMeta VolumeMeta
```

Comment:
I have added gorm:"foreignKey:VolumeMetaID" as this should reference a VolumeMeta- correct?


```go
VolumeMeta VolumeMeta `gorm:"foreignKey:VolumeMetaID"`
```

Reply:
This is redundant, it will work without this line.

Reply:
Because if you introspect table you can see it works without this extra annotation:
<img width="615" alt="image" src="https://github.com/Dynatrace/dynatrace-operator/assets/1374633/1c54b9cb-289e-4c9f-b4d4-a1e0d809de9a">

Reply:
true... I just like the explicit- in case of renaming it could save us a headache